images/techdocs/tests/test_image.py

```diff
@@ -119,3 +119,25 @@ def test_build(
         b"info: Successfully generated docs from /content into /content/site using local mkdocs"
         in actual_output
     )
+
+@pytest.mark.parametrize(
+    "cli_command,expected_ouput",
+    [
+        ("techdocs-cli --version", b"1."),
+        ("markdownlint --version", b"0.32."),
+        ("vale --version", b"vale version v2."),
+        ("yq --version", b"yq (https://github.com/mikefarah/yq/) version 4."),
```

Comment:
```suggestion
        ("yq --version", b"yq (https://github.com/mikefarah/yq/) version v4."),
```

I had to add a `v` in the version to `yq` in the test above after release `v4.30.4`

https://github.com/mikefarah/yq/issues/1434

Reply:
This test is passing on the main branch and for this PR, can change it in another PR together with updating yq.